qt/qssl_wolf/tst_wolfssl.cpp

```diff
@@ -805,17 +805,22 @@ void tst_QSslWolfSSL::peerCertificateChainToWWWpQTpIO()
     *            ISRG Root X2
     *            ISRG Root X1
     *            DST Root CA X3
+    * This becomes obsolate as of 2024.07.17. Now, www.qt.io returns 3 chains
```

Comment:
Isn't there a better way to fix below by checking the correct chain?

Reply:
Checking chain contents instead of skipping it